irb.gemspec

```diff
@@ -16,6 +16,11 @@ Gem::Specification.new do |spec|
   spec.homepage      = "https://github.com/ruby/irb"
   spec.licenses      = ["Ruby", "BSD-2-Clause"]
 
+  spec.metadata["homepage_uri"] = spec.homepage
+  spec.metadata["source_code_uri"] = spec.homepage
+  spec.metadata["documentation_uri"] = spec.homepage
+  spec.metadata["changelog_uri"] = "#{spec.homepage}/releases"
```

Comment:
I figure we don't want to maintain a `CHANGELOG` file yet so let's just use the release generated by GH for now, which will be enhanced after #531.